modules/SofaGeneralEngine/src/SofaGeneralEngine/config.h.in

```diff
@@ -24,7 +24,7 @@
 #include <sofa/config.h>
 #include <sofa/config/sharedlibrary_defines.h>
 
-#define SOFAGENERALENGINE_HAVE_SOFA_GL @SOFAGENERALENGINE_HAVE_SOFA_GL@
+#define SOFAGENERALENGINE_HAVE_SOFA_GL_COMPONENT_ENGINE @SOFAGENERALENGINE_HAVE_SOFA_GL_COMPONENT_ENGINE@
```

Comment:
same here, people were (maybe) using SOFAGENERALENGINE_HAVE_SOFA_GL variable. But if it is is renamed, this is new; so the user will have an error.